collectors/cgroups.plugin/cgroup-name.sh

```diff
@@ -72,7 +88,9 @@ function docker_like_get_name_api() {
     info "Running API command: curl \"${host}${path}\""
     JSON=$(curl -sS "${host}${path}")
   fi
-  NAME=$(echo "${JSON}" | jq -r .Name,.Config.Hostname | grep -v null | head -n1 | sed 's|^/||')
+  if OUTPUT=$(echo "${JSON}" | jq -r '.Config.Env[],"CONT_NAME=\(.Name)"') && [ -n "$OUTPUT" ]; then
+    NAME="$(parse_docker_like_inspect_output "$OUTPUT")"
```

Comment:
- I think that this won't work because you don't get newlines in this output, but I'll test now.
- If we're essentially just doing docker inspect both times, we might as well not use `jq` at all. I hadn't realized it's the exact same output.

Reply:
It works.

```
deb-work ~ $ curl -s --unix /var/run/docker.sock http://localhost/containers/4f6eb6a45750/json | jq -r '.Config.Env[],"CONT_NAME=\(.Name)"'
NETDATA_LISTENER_PORT=20000
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
NETDATA_OFFICIAL_IMAGE=true
DOCKER_GRP=netdata
DOCKER_USR=netdata
CONT_NAME=/nd
```

> If we're essentially just doing docker inspect both times, we might as well not use jq at all

We can't when using REST API.

Reply:
It's because `jq` does the formatting into new lines. This is annoying, having this dependency just to put the newlines there.

Reply:
Ah, you mean we can just `grep` the output? I think that is possible - anyway, this function is used when Netdata is running in a Docker container and there we have `jq` installed - not a problem.

Reply:
I see, we need the formatting too, for the `CONT_NAME`. Unfortunate, but the alternative would be to do a hack, so it's better this way.

Reply:
I tested both functions:

- `docker_like_get_name_command()` - Netdata running on the host.
- `docker_like_get_name_api() ` - Netdata running in a Docker container.